resources/views/themes/default1/user/client/index.blade.php

```diff
@@ -25,7 +26,7 @@
                     <h3 class="card-title">Advance Search</h3>
 
                     <div class="card-tools">
-                        <button type="button" class="btn btn-tool" data-card-widget="collapse" data-toggle="tooltip" title="Collapse">
+                        <button type="button" class="btn btn-tool" data-card-widget="collapse" data-toggle="tooltip" title="Fillters">
```

Comment:
Check Filter spelling.
Also it should be Expand/Collapse here since we are using the +/- button

Reply:
Fixed